app/components/card.tsx

```diff
@@ -9,14 +10,16 @@ interface CardProps {
     name: string;
     age: number;
     job: Job;
+    title: string;
```

Comment:
**Suggestion:** The `title` prop should have a default value of an empty string ('').
```suggestion
    title: string = '';
```